media/Circletracer/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <!--
+					Use a content security policy to only allow loading images from https or from our extension directory,
+					and only allow scripts that have a specific nonce.
+				-->
+    <meta http-equiv="Content-Security-Policy"
+        content="default-src 'none'; style-src _webview.cspSource; img-src _webview.cspSource https: http: data: blob:; script-src 'unsafe-inline' data: blob: http: https:; connect-src _webview.cspSource https:">
```

Comment:
I think we should use our `nonce` to increase security.
Although the `nonce` value is used now, it is not being reviewed in the security policy.

Reply:
Is there any case that we need `http`, `blob` or other things except accessing to `https://d3js.org/d3.v5.min.js`?

Reply:
I change security policy setting. :)